src/layer.rs

```diff
@@ -250,6 +255,11 @@ impl Default for LayerSet {
     }
 }
 
+#[cfg(not(feature = "druid"))]
+type MaybeArcGlyph = Glyph;
+#[cfg(feature = "druid")]
+type MaybeArcGlyph = Arc<Glyph>;
+
```

Comment:
instead of getting fancy here, let's just put the `entry` method behind a `#[cfg(not(feature = "druid"))]`? the druid stuff is pretty legacy at this point, and if in the future I need this feature I can deal with it then. :)

Reply:
Should we do a pass to remove druid code then?